packages/channels/package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@slack/bolt": "^3.9.0",
     "axios": "^0.25.0",
+    "botbuilder": "^4.15.0",
     "cli-color": "^2.0.1",
```

Comment:
We've upgraded a whopping 1 minor version here (from 4.14.0). I think I may have updated this channel before.